### PR TITLE
[ESLint Fixes] moving to @stylistic plugin + fixes

### PR DESCRIPTION
**Pull request by jackyalbo**
### Explain the changes
1. In addition to prior PR, seems like moving to stylistic plugin is pretty straightforward

### Issues: Fixed #xxx / Gap #xxx
1. 

### Testing Instructions:
1. 

- [ ] Doc added/updated
- [ ] Tests added

### CODE REVIEW

**Review by liranmauda — APPROVED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ module.exports = {
 
     plugins: [
         // eslint-plugin-header validates copyright header
+        '@stylistic/js',
```
Comment (shirady) [inline .eslintrc.js:25]: Please move this after the 'header' (the comment above refers to it).

**Review by shirady — APPROVED** [review]
LGTM - minor comment

**Comment by shirady** [comment]
@liranmauda should this PR separated into a couple of commits?
1. Adding the plugin + and it's changes.
2. Fixing existing errors (in file `health.js`).

**Comment by liranmauda** [comment]
> @liranmauda should this PR separated into a couple of commits?
> 
> 1. Adding the plugin + and it's changes.
> 2. Fixing existing errors (in file `health.js`).


Basically, yes.
This case is where we will want to backport only the eslint fixes to a branch where we don't have the `health.js`

I missed it, Great catch.
as this PR is merged, if we need to backport we will need to drop this conflict.

**Pull request merged**